app/models/utils.py

```diff
@@ -1,9 +1,29 @@
+from typing import Tuple
+
 from app.enums.care_day_type import CareDayType
 from app.models.payment_rate import PaymentRate
 
 
-def get_care_day_cost(day_type: CareDayType, provider_id: str, child_id: str) -> int:
+def get_care_day_cost(
+    day_type: CareDayType, provider_id: str, child_id: str, allocation_remaining: int
+) -> Tuple[int, int]:
```

Comment:
nit: No need to import the type
```suggestion
) -> tuple[int, int]:
```